packages/cli/src/eventbus/eventBusRoutes.ts

```diff
@@ -0,0 +1,236 @@
+	return o.url !== undefined;
+};
+
+const isMessageEventBusDestinationOptions = (
+	candidate: unknown,
+): candidate is MessageEventBusDestinationOptions => {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+	const o = candidate as MessageEventBusDestinationOptions;
+	if (!o) return false;
+	return o.__type !== undefined;
+};
+
+// ----------------------------------------
+// Events
+// ----------------------------------------
+eventBusRouter.get(
+	`/event`,
+	ResponseHelper.send(async (req: express.Request): Promise<any> => {
+		if (isWithQueryString(req.query)) {
+			switch (req.query.query as EventMessageReturnMode) {
+				case 'sent':
+					return eventBus.getEventsSent();
+				case 'unsent':
+					return eventBus.getEventsUnsent();
+				case 'all':
+				default:
+			}
+		}
+		return eventBus.getEvents();
+	}),
+);
+
+eventBusRouter.post(
+	`/event`,
+	ResponseHelper.send(async (req: express.Request): Promise<any> => {
+		if (isEventMessageOptions(req.body)) {
+			let msg;
+			switch (req.body.__type) {
+				case EventMessageTypeNames.workflow:
+					msg = new EventMessageWorkflow(req.body as EventMessageWorkflowOptions);
+					break;
+				case EventMessageTypeNames.audit:
+					msg = new EventMessageAudit(req.body as EventMessageAuditOptions);
+					break;
+				case EventMessageTypeNames.generic:
+				default:
+					msg = new EventMessageGeneric(req.body);
+			}
+			await eventBus.send(msg);
+		} else {
+			throw new BadRequestError(
+				'Body is not a serialized EventMessage or eventName does not match format {namespace}.{domain}.{event}',
+			);
+		}
+	}),
+);
+
+eventBusRouter.post(
+	`/event/addmany/:count`,
+	ResponseHelper.send(async (req: express.Request): Promise<any> => {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+		if (isEventMessageOptionsWithType(req.body, EventMessageTypeNames.workflow)) {
+			const count: number = parseInt(req.params.count) ?? 100;
+			for (let i = 0; i < count; i++) {
+				const msg = new EventMessageWorkflow(req.body as EventMessageWorkflowOptions);
+				msg.setPayload({
+					id: i,
+					msg: 'REST test',
+				});
+				await eventBus.send(msg);
+			}
+		} else {
+			throw new BadRequestError(
+				'Body is not a serialized EventMessage or eventName does not match format {namespace}.{domain}.{event}',
+			);
+		}
+	}),
+);
+
+// ----------------------------------------
+// Destinations
+// ----------------------------------------
+
+eventBusRouter.get(
+	`/destination`,
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	ResponseHelper.send(async (req: express.Request, res: express.Response): Promise<any> => {
+		let result = [];
+		if (isWithIdString(req.query)) {
+			result = await eventBus.findDestination(req.query.id);
+		} else {
+			result = await eventBus.findDestination();
+		}
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+		return result;
+	}),
+);
+
+eventBusRouter.post(
```

Comment:
Shouldn't those endpoints be protected to be accessible by instance owner only?

Reply:
will change both back and frontend to be only available to owners (for change actions, reading available for all)